templates/coordinate/enrollments/edit/form.hbs

```diff
@@ -0,0 +1,62 @@
+<div class="row justify-content-center no-gutters">
+    {{#with data.enrollments_by_pk}}
+    <div class="card text-dark col-sm-11 col-md-9 col-lg-8">
+        <div class="card-header">
+           <h1 class="card-title">
+                Edit {{user.first_name}} {{user.last_name}}'s enrollment in {{semester.title}}
+           </h1>
+        </div>
+        <div class="card-body">
+            <form onsubmit="return alert('Changes Saved!');" method="post">
+                <div class="form-row">
+                    <label for="lead-select">Is Project Lead:</label>
+                    <select class="form-control" id="lead-select", name="lead" required>
+                        <option value="true" {{#if is_project_lead}} selected {{/if}}> true </option>
+                        <option value="false" {{#unless is_project_lead}} selected {{/unless}}> false </option>
+                    </select>
+                </div>
+                <div class="form-row">
+                    <label for="coordinator-select">Is Coordinator:</label>
+                    <select class="form-control" id="coordinator-select", name="coordinator" required>
+                        <option value="true" {{#if is_coordinator}} selected {{/if}}> true </option>
+                        <option value="false" {{#unless is_coordinator}} selected {{/unless}}> false </option>
+                    </select>
+                </div>
+                <div class="form-row">
+                    <label for="pay-select">Is for Pay:</label>
+                    <select class="form-control" id="pay-select", name="pay" required>
+                        <option value="true" {{#if is_for_pay}} selected {{/if}}> true </option>
+                        <option value="false" {{#unless is_for_pay}} selected {{/unless}}> false </option>
+                    </select>
+                </div>
+                <div class="form-row">
+                    <label for="mid-year-grade">Mid Year Grade:</label>
+                    <input id="mid-year-grade" type="number" step="0.01" name="mid_grade" value="{{mid_year_grade}}"/>
+                </div>
+                <div class="form-row">
+                    <label for="final_grade">Final Grade:</label>
+                    <input id="final_grade" type="number" step="0.01" name="final_grade" value="{{final_grade}}"/>
```

Comment:
Could you add some classes to make this match the other fields?
A good example is at https://getbootstrap.com/docs/4.0/components/forms/#horizontal-form